doc/codi.txt

```diff
@@ -215,10 +215,12 @@ g:codi#interpreters
              Default value is {}.
 
                                                     *g:codi#environment_flags*
```

Comment:
Please also change this header, and make sure that other reference (e.g. table of contents) are updated.